npm/test/basics.js

```diff
@@ -6,20 +6,18 @@
 import assert from "node:assert";
 import { test } from "node:test";
 import { log } from "../dist/log.js";
-import { getCompiler, getCompilerWorker } from "../dist/main.js";
-import { QscEventTarget } from "../dist/events.js";
+import {
+  getCompiler,
+  getCompilerWorker,
+  getLanguageService,
+  getLanguageServiceWorker,
+} from "../dist/main.js";
+import { QscEventTarget } from "../dist/compiler/events.js";
 import { getKata } from "../dist/katas.js";
 import samples from "../dist/samples.generated.js";
 
 log.setLogLevel("warn");
 
-/**
- *
- * @param {string} code
- * @param {string} expr
- * @param {boolean} useWorker
- * @returns {Promise<import("../dist/common.js").ShotResult>}
- */
```

Comment:
Why remove these comments? The JS file has ts-check on it, so this avoids all the params being 'any' and improves type checking.

Reply:
Oh, I didn't realize these were for typechecking. I thought they were accidentally generated JSDoc comments. I'll revert it!